public/app/modules/shared_services/dataservice.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Created by tremaine on 1/19/16.
+ */
+(function(){
+    angular
+        .module('dataservice',[])
+        .factory('dataService', dataService);
+
+    dataService.$inject = ['$http'];
+    function dataService($http){
+        var service = {
+            getPosts : getPosts
+        };
+
+        return service;
+
+        function getPosts(){
```

Comment:
@tremainebuchanan if we are using the angular framework, let's use it properly.  Don't use `$http.get()` - use the factory method angular specifies.